SPECS/python-zipp/python-zipp.spec

```diff
@@ -3,19 +3,20 @@
 Summary:        Backport of pathlib-compatible object wrapper for zip files
 Name:           python-%{pypi_name}
 Version:        3.17.0
-Release:        1%{?dist}
+Release:        2%{?dist}
 License:        MIT
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 URL:            https://github.com/jaraco/zipp
-Source0:        %{pypi_source}
```

Comment:
The `pypi_source` macro should work correctly, if we've got everything right. Can you check, if we can revert the change to this one line?

Reply:
where is this set?

Reply:
ok, I see now that many other specs are also using this, and it appears to work. I've reverted this change